packages/web-components/src/components/va-table/va-table.tsx

```diff
@@ -35,6 +35,11 @@ export class VaTable {
    * If uswds is true, the type of table
    */
   @Prop() tableType?: 'borderless' = 'borderless';
+
+  /**
+   * If true convert to a stacked table when screen size is small
+   */
+  @Prop() stacked?: boolean = false;
```

Comment:
@ataker is this boolean to set stacked at xsm sizes, or to force the stack view at all times?
@rsmithadhoc should the default not be true for accessibility reasons?
Would there be a use case to force stacked view at larger screen sizes?

Reply:
@rmessina1010 I would say for accessibility, the version that fits on the screen is the one that should be rendered. So I think we would want it to be `true`, so that happens by default.

Otherwise, if this was `false` every table instance would be non-stacked by default, no matter the screen size, right? So that may lead to things getting cut off unexpectedly on smaller screens.

Reply:
Updated to on by default. Changed the stories in storybook to reflect the default behavior.